tests/braille-specs/da-dk-g28.yaml

```diff
@@ -224,6 +224,105 @@ tests:
   - ['þ', '⠠⢅']
   - ['ÿ', '⠠⢾']
 
+# Misc Unicode chars
+# For each accented letter, the first occurring instance is chosen for back-translation.
+# There are no rules about this in the specs of Danish Braille,
```

Comment:
@BueVest You included this notice for g18 and g28. Isn't it also true for parts of the 6-dot tables (I saw a comment about this in da-dk-6miscChars.cti)?